daml-assistant/integration-tests/src/Main.hs

```diff
@@ -278,31 +279,31 @@ packagingTests tmpDir = testGroup "packaging"
             ]
         writeFileUTF8 (projectB </> "daml.yaml") $ unlines
             [ "sdk-version: " <> sdkVersion
-            , "version: \"1.0\""
-            , "name: b"
+            , "version: \"2.0\""
+            , "name: a"
             , "source: daml/Main.daml"
             , "exposed-modules: [Main]"
             , "dependencies:"
             , "  - daml-prim"
             , "  - daml-stdlib"
             ]
-        withCurrentDirectory projectB $ callCommandQuiet $ "daml build --output " <> distDir </> "b.dar"
-        assertBool "b.dar was not created." =<< doesFileExist bDar
-        step "Creating migration project"
-        callCommandQuiet $ unwords ["daml", "migrate", projectMigrate, "daml/Main.daml", aDar, bDar]
+        withCurrentDirectory projectB $ callCommandQuiet "daml build"
+        assertBool "a-2.0.dar was not created." =<< doesFileExist bDar
+        step "Creating upgrade project"
+        callCommandQuiet $ unwords ["daml", "migrate", projectUpgrade, "daml/Main.daml", aDar, bDar]
         step "Build migration project"
-        withCurrentDirectory projectMigrate $ callCommandQuiet $ "daml build --output " <> distDir </> "c.dar"
+        withCurrentDirectory projectUpgrade $ callCommandQuiet "daml build --init-package-db=no --package \'(\"a-1.0\", [(\"Main\", \"MainA\")])\' --package \'(\"a-2.0\", [(\"Main\", \"MainB\")])\'"
```

Comment:
Why do you need `--init-package-db=no`? Is this an optimization because `daml migrate` already takes care off this or a requirement because running `daml init` again will mess things up in some way?

Reply:
yes, `migrate` takes care of setting up a sensible package database. `daml init` would mess things up.

Reply:
In that case, should we turn `migrate + build` into a single command (with a --no-build option for debugging)? It seems a bit unintuitive that`daml build` doesn’t work without special flags.

Reply:
I now changed it, so that the migration project contains a `build.sh` that you can run. Generating the migration project and building it are two different processes. That the build will fail, will be a common thing, since the newer package might contain non-isomorphic contract templates. So I would not couple the two commands and instead, provide an easy way to try the build again once changes have been made manually to the source.

Reply:
Sounds good, thanks!